build/zip_bundle.gni

```diff
@@ -57,6 +91,16 @@ template("zip_bundle") {
       ]
       sources += [ input.source ]
     }
+    if (invoker.encode_entitlements) {
```

Comment:
Can you please add docs for the entitlements parameters. E.g. are they expecting the names with directories or just file names? e.g. for a file a/b/c.txt does it need to include path info as a/b/c.txt or just c.txt?